platform/solr/solr-factory/src/main/java/org/codice/solr/client/solrj/SolrClient.java

```diff
@@ -766,6 +767,16 @@ public UpdateResponse deleteByQuery(String collection, String query, int commitW
   public UpdateResponse deleteByQuery(String query, int commitWithinMs)
       throws SolrServerException, IOException;
 
+  /**
+   * Issues a ping request to check if the collection's replicas are alive
+   *
+   * @param collection the Solr collection to ping
+   * @return a {@link SolrPingResponse} containing the response from the server
+   * @throws IOException If there is a low-level I/O error
```

Comment:
consistency is king 👑 
```suggestion
   * @throws IOException if there is a low-level I/O error
```